document-portal/document-portal-fuse.c

```diff
@@ -1753,16 +1753,8 @@ xdp_fuse_create (fuse_req_t             req,
         }
 
```

Comment:
I think the commit message was meant to say "...are not really *under* our control..."

Reply:
fixed